browser-bunyan/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
Looking at the [error message](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/188708952#L1489), that was a lint error. Completely unrelated to `export *`. Could you try it again and run the linter locally to make sure you have no lint errors?